integration/fuse/src/main/java/alluxio/fuse/file/FuseFileInStream.java

```diff
@@ -78,7 +79,8 @@ public static FuseFileInStream create(FileSystem fileSystem, FuseReadWriteLockMa
       }
 
       try {
-        FileInStream is = fileSystem.openFile(uri);
+        FileInStream is = fileSystem.openFile(status.get(),
```

Comment:
I think the real fix is here: the URIStatus is already retrieved from Master.
So we can avoid an extra operation by passing the result to .openFile().

Reply:
Yeah, ideally each client only gets one status  from source of truth (UFS or master) for the whole read process.
The current problem is that without metadata cache enabled, for a single read process we will ask for file metadata for multiple times. We didn't store the file status in client for read/write process.